templates/registries.yaml.j2

```
@@ -0,0 +1 @@
+{{ k3s_registries | to_nice_yaml }}
```

```suggestion
---

{{ k3s_registries | to_nice_yaml }}

```

YAML header, newline at end of file.